src/main/webapp/app/entities/result/result.component.ts

```diff
@@ -36,10 +30,8 @@ export class ResultComponent implements OnInit, OnChanges, OnDestroy {
     @Input() short = false;
     @Input() result: Result;
     @Input() showUngradedResults: boolean;
-    @Output() newResult = new EventEmitter<object>();
+    @Input() showBadge = false;
```

Comment:
I think it would be better to call it something like "showGradedBadge".

Reply:
renamed it in https://github.com/ls1intum/ArTEMiS/pull/448